tests/integration/test_remoted/test_active_response/test_active_response_send_ar.py

```diff
@@ -0,0 +1,93 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import os
+import pytest
+import time
+
+import wazuh_testing.remote as remote
+import wazuh_testing.tools.agent_simulator as ag
+from wazuh_testing.tools.configuration import load_wazuh_configurations
+from wazuh_testing.tools.sockets import send_active_response_message
+from wazuh_testing import UDP, TCP, TCP_UDP
```

Comment:
```suggestion
import wazuh_testing.remote as remote
import wazuh_testing.tools.agent_simulator as ag
from wazuh_testing import UDP, TCP, TCP_UDP
from wazuh_testing.tools.configuration import load_wazuh_configurations
from wazuh_testing.tools.sockets import send_active_response_message
```

Reply:
Done in 3cc7447ababd79cb7c49087dde0a2fbfe5cec5e1